lib/elixir/test/elixir/task/supervisor_test.exs

```diff
@@ -221,7 +221,7 @@ defmodule Task.SupervisorTest do
       assert supervisor
              |> Task.Supervisor.async_stream(1..4, &exit/1, @opts)
              |> Enum.to_list ==
-             [exit: 1, exit: 2, exit: 3, exit: 4]
+             [{:exit, 1, 1}, {:exit, 2, 2}, {:exit, 3, 3}, {:exit, 4, 4}]
```

Comment:
I think we can improve one of these tests by doing something like

```elixir
Task.Supervisor.async_stream(1..4, &exit(Integer.to_string(&1)), @opts)
```

so that the exit reason and the original element are different. Wdyt?